xml/osc_building.xml

```diff
@@ -196,5 +195,192 @@ The buildroot was: /var/tmp/build-root/openSUSE_Tumbleweed-x86_64</screen>
+   </listitem>
+  </itemizedlist>
+  <para>To build a new &obs.ai; from RPM packages, proceed as follows:</para>
+  <procedure>
+   <title>Building &obs.ai; from RPM Packages</title>
+   <step>
+    <para>
+     Start with building RPM packages as described in <xref
+      linkend="sec.obs.building.spec2rpm"/>. Make sure the packages has
+     been built successfully.
+    </para>
+   </step>
+   <step>
+    <para>
+     Prepare your project to support &obs.ai;. In this example, we
+     use the home project <uri>&obshome1;</uri>:
+    </para>
+    <substeps>
+     <step>
+      <para>
+       To open the meta configuration, run:
+      </para>
+      <screen>&prompt.user;<command>osc</command> meta prj -e &obshome1;</screen>
+      <para>
+       <remark>toms 2017-09-05: Maybe this step should be explained in
+       a separate section/procedure?</remark>
+       If you want to use the Web interface of your &obsa; instance to do
+       the same, go to your home project and click <menuchoice>
+        <guimenu>Advanced</guimenu>
+        <guimenu>Meta</guimenu>
+       </menuchoice>.
+      </para>
+     </step>
+     <step>
+      <para>
+       To enable &obs.ai; support, add the following XML snippet into the editor:
+      </para>
+      <screen language="xml">&lt;repository name="AppImage">
+  &lt;path project="&obshome1;" repository="<replaceable>PROJECT</replaceable>"/>
+  &lt;path project="OBS:AppImage" repository="AppImage"
+  &lt;arch>x86_64&lt;/arch>
+  &lt;!-- &lt;arch>i586&lt;/arch> -->
+&lt;/repository></screen>
+     </step>
+     <step>
+      <para>
+       Adapt the <replaceable>PROJECT</replaceable> placeholder and the
+       architecture.
+      </para>
+     </step>
+     <step>
+      <para>
+       To save the new configuration, use <keycombo>
+        <keycap>:</keycap>
+        <keycap>x</keycap>
+       </keycombo> or leave the editor with <keycombo>
+        <keycap>:</keycap>
+        <keycap>q</keycap>
+        <keycap>!</keycap>
+       </keycombo>.
+      </para>
+     </step>
+    </substeps>
+   </step>
+   <step>
+    <para>
+     Create a new <filename>appimage.yml</filename> with the following
+     contents:
+    </para>
+    <example>
+     <title>&obs.ai; from RPM Files (<filename>appimage.yml</filename>)</title>
+     <screen language="yaml">app: <replaceable>APPNAME</replaceable> <co xml:id="co.obs.building.ai.app"/>
+
+build: <co xml:id="co.obs.building.ai.build"/>
+  packages:
+    - linuxdeployqt
+
+ingredients: <co xml:id="co.obs.building.ai.ingredients"/>
+  packages:
+    - <replaceable>MY_RPM_PACKAGE</replaceable>
+
+script: <co xml:id="co.obs.building.ai.script"/>
+  # instructions
+     </screen>
+     <calloutlist>
+      <callout arearefs="co.obs.building.ai.app">
+       <para>
+        <remark>toms 2017-09-05: any restrictions in naming?</remark>
+        Contains the name of &obs.ai; app. You can use any name you want,
+        but it is recommended to use lowercase names.
```

Comment:
"it is recommended to use"->"we recommend using"

Reply:
Fixed.